augur/curate/__init__.py

```diff
@@ -213,14 +217,17 @@ def run(args):
             input files can be provided via the command line options `--metadata` and `--fasta`.
             See the command's help message for more details."""))
 
+    # Get the name of the subcmd being run
+    subcmd_name = getattr(args, SUBCOMMAND_ATTRIBUTE, None).__name__.split(".")[-1].replace("_","-")
```

Comment:
The subcommand name should be directly accessible because we specify `dest="subcommand"` in 

https://github.com/nextstrain/augur/blob/08e35eec25d99339e23b4298d5685577fee4af67/augur/curate/__init__.py#L112-L113

```suggestion
    subcmd_name = args.subcommand
```

Reply:
ah, thanks, that felt like i was going the long way 'round... force-pushed.